src/plugins/orangehrmAdminPlugin/config/supported_languages.yml

```diff
@@ -10,6 +10,7 @@ languages:
   6: {key: 'de', value: 'German - Deutsch'}
   7: {key: 'es', value: 'Spanish - Español'}
   8: {key: 'es_CR', value: 'Spanish (Costa Rica) - Español (Costa Rica)'}
+  9: {key: 'SI', value: 'Slovenian'} 
```

Comment:
Is capitalized key here correct or should it be `si`?

Reply:
tested with SI, it worked, tested with si, didn't work